plugins/lua/stockpiles.lua

```diff
@@ -428,10 +428,10 @@ end
 StockpilesOverlay = defclass(StockpilesOverlay, overlay.OverlayWidget)
 StockpilesOverlay.ATTRS{
     desc='Shows a panel when a stockpile is selected for stockpile automation.',
-    default_pos={x=24, y=-6},
+    default_pos={x=5, y=-11},
```

Comment:
This position covers the categories. I think it might be better closer to where it was.
![image](https://github.com/DFHack/dfhack/assets/977482/adecf78c-97aa-45f6-87db-18ed7074293d)

however, with the increased height, in its old position, it now completely covers the "add stockpile" button, which is also not good.

what is the least bad option here? should we keep everything on two lines and make the widget even wider?

Reply:
Oh, I wanted to ask because I saw that other open PR about it, should I call it ‘claim’ instead of unforbid? I take it that’s the new ingame language used?

Reply:
Or maybe it could stay open on the stockpile category settings screen but move locations since the stockpile button is already covered?

Any tips on resolution testing? I had tried 1920x1080 and 1024x768 and didn’t have the overlap shown in your screenshot, which appears to be 1920x1200? More vertical pixels and we have less vertical space! haha - or am I overlooking other UI scaling features?

Reply:
I find the best way to test is to have DF in windowed mode and then dynamically resize the window from very large to the smallest size (where DF starts scaling the graphics instead of reducing the column/row count)